config/owasp/suppressions.xml

```diff
@@ -24,5 +24,6 @@
     <cve>CVE-2022-45143</cve>
     <cve>CVE-2021-4235</cve>
     <cve>CVE-2022-3064</cve>
+    <cve>CVE-2022-45688</cve>
```

Comment:
How about create a Jira  ticket and add the reference in here, so we can fix it later? Feel free to assign that Jira to me.
```
<suppress>
     <notes>Temporary Suppression 
      CVE-2022-45688 refer [Ticket]
      </notes>
<cve>CVE-2022-45688</cve>
</suppress>
```